src/client/utils/tracking/gtm.ts

```diff
@@ -132,6 +133,10 @@ export const trackOfferGTM = (
   const grossPrice = Math.round(Number(offerData.cost.monthlyGross.amount))
   const netPrice = Math.round(Number(offerData.cost.monthlyNet.amount))
 
+  const currentInsurer = offerData.quotes[0].currentInsurer?.id ?? undefined
+  const externalInsuranceData = getExternalInsuranceDataFromGQLCache(
+    offerData.quotes[0]?.dataCollectionId,
+  )
```

Comment:
We should use our selectors for this - or is it not possible in this case?

Reply:
i've went ahead and made a new one